core-web/libs/template-builder/src/lib/components/template-builder/components/add-style-classes-dialog/add-style-classes-dialog.component.scss

```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+:host {
+    width: 33.875rem;
+    display: block;
+}
+
+.dialog__auto-complete-container {
+    padding: 0.375rem $spacing-7 $spacing-1;
```

Comment:
Why `0.375rem`?

Reply:
Changing to 0 or 0.5 rem, depends on how it looks